server/py/framework/db/sqldb/models.py

```diff
@@ -968,6 +968,17 @@ class ModelEndpoint(Base, mlrun.utils.db.HasStruct):
         def get_identifier_string(self) -> str:
             return f"{self.project}_{self.name}_{self.created}"
 
+    class SystemMetadata(Base, mlrun.utils.db.BaseModel):
+        __tablename__ = "system_metadata"
+        __table_args__ = (UniqueConstraint("key", name="_system_metadata_uc"),)
+
+        id = Column(Integer, primary_key=True)
+        key = Column(String(255, collation=SQLTypesUtil.collation()), nullable=False)
+        value = Column(String(255, collation=SQLTypesUtil.collation()))
```

Comment:
In which cases value supposed to be nullable?

Reply:
@Yacouby then why do we need to keep key if value is not there?

Reply:
on a second thought if adding a boolean key, then the value should be True/False, then anyway it should not be null.. updated